Add "thirdcircle" option to allow narrowed hemicycles

RichardMN opened the pull request:
`calc_coordinates` can now handle segment values other than 1 and 0.5, making a symmetrical narrowed cycle.
`parliament_data` now takes "`thirdcircle`" as an option to create a 1/3 pie cycle. `parliament_data` now also gives a more informative error for non-null incorrect `type` 
First vignette has an example of such a parliament, using data from the Australian senate.
Documentation has been rebuilt (but this was largely a function of running checks).

zmeers commented on R/parliament_data.R:
```diff
@@ -29,6 +29,7 @@ parliament_data <- function(election_data = NULL,
                             type = c(
                               "horseshoe",
                               "semicircle",
+                              "thirdcircle",
```
Should it be "thirdcircle" or "third_circle"?

RichardMN replied:
Could it be both? I'm inclined to allow more synonymous options - I have another branch where I've put in "hemicycle" for "semicircle" since this is a common term in French to describe the format of a such a parliament.

zmeers reviewed (approved):
LGTM - minor comment re: wording. Please see below.

zmeers commented:
Apologies @RichardMN I forgot to tag you in my review - feel free to merge if you like. It's a substantial enough contribution to add your name to the author list as well, if you want.

RichardMN commented:
Sorry for the delay in responding on this. I've fixed the error warning for a mis-typed parliament type and added in a synonym. I think there may be an elegant way of handling a list of possible options, writing it once and having it show up in both places, but I've not written it.
I don't think I have permissions to merge this. @zmeers - if you're ok with this, please merge it in for me.
